packages/tools/src/tools/segmentation/ThresholdPreviewTool.ts

```diff
@@ -0,0 +1,235 @@
+import {
+  activeSegmentation,
+  segmentIndex as segmentIndexController,
+  config as segmentationConfig,
+  state as segmentationState,
+} from '../../stateManagement/segmentation';
+import { getSegmentation } from '../../stateManagement/segmentation/segmentationState';
+import { PublicToolProps, ToolProps } from 'tools/src/types';
+import { BaseTool } from '../base';
+import { fillInsideCube } from './strategies/fillCube';
+import { cache } from '@cornerstonejs/core';
+
+import { getToolGroup } from '../../store/ToolGroupManager';
+
+import { getEnabledElementByIds } from '@cornerstonejs/core';
+import { LabelmapSegmentationData } from 'tools/src/types/LabelmapTypes';
```

Comment:
should not import like this, probably your autocomplete

Reply:
Done